go-controller/pkg/ovn/hybrid.go

```diff
@@ -134,14 +136,18 @@ func (oc *DefaultNetworkController) handleHybridOverlayPort(node *kapi.Node, ann
 	return nil
 }
 
-func (oc *DefaultNetworkController) deleteHybridOverlayPort(node *kapi.Node) {
+func (oc *DefaultNetworkController) deleteHybridOverlayPort(node *kapi.Node) error {
 	klog.Infof("Removing node %s hybrid overlay port", node.Name)
 	portName := util.GetHybridOverlayPortName(node.Name)
 	lsp := nbdb.LogicalSwitchPort{Name: portName}
 	sw := nbdb.LogicalSwitch{Name: node.Name}
-	if err := libovsdbops.DeleteLogicalSwitchPorts(oc.nbClient, &sw, &lsp); err != nil {
-		klog.Errorf("Failed deleting hybrind overlay port %s for node %s err: %v", portName, node.Name, err)
+	if err := libovsdbops.DeleteLogicalSwitchPorts(oc.nbClient, &sw, &lsp); err != nil && !errors.Is(err, libovsdbclient.ErrNotFound) {
```

Comment:
nit: `DeleteLogicalSwitchPorts` should not, by convention, return an `libovsdbclient.ErrNotFound` error
If what you want to remove is not found, it will not fail

Reply:
removed the check for the returned error